Content.Server/Defusable/Systems/DefusableSystem.cs

```diff
@@ -0,0 +1,228 @@
+    [Dependency] private readonly TriggerSystem _trigger = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+
+    /// <inheritdoc/>
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<DefusableComponent, ExaminedEvent>(OnExamine);
+        SubscribeLocalEvent<DefusableComponent, GetVerbsEvent<AlternativeVerb>>(OnGetAltVerbs);
+        SubscribeLocalEvent<DefusableComponent, AnchorAttemptEvent>(OnAnchorAttempt);
+        SubscribeLocalEvent<DefusableComponent, UnanchorAttemptEvent>(OnUnanchorAttempt);
+    }
+
+    #region Subscribed Events
+
+    /// <summary>
+    ///     Adds a verb allowing for the bomb to be started easily.
+    /// </summary>
+    private void OnGetAltVerbs(EntityUid uid, DefusableComponent comp, GetVerbsEvent<AlternativeVerb> args)
+    {
+        if (!args.CanInteract || !args.CanAccess)
+            return;
+
+        args.Verbs.Add(new AlternativeVerb()
+        {
+            Text = Loc.GetString("defusable-verb-begin"),
+            Disabled = comp.BombLive && comp.BombUsable,
+            Priority = 10,
+            Act = () =>
+            {
+                TryStartCountdown(uid, comp);
+            }
+        });
+    }
+
+    private void OnExamine(EntityUid uid, DefusableComponent comp, ExaminedEvent args)
+    {
+        if (!args.IsInDetailsRange)
+            return;
+
+        if (!comp.BombUsable)
+        {
+            args.PushMarkup(Loc.GetString("defusable-examine-defused", ("name", uid)));
+        }
+        else if (comp.BombLive && TryComp<ActiveTimerTriggerComponent>(uid, out var activeComp))
+        {
+            if (comp.BombDisplayTime)
+            {
+                args.PushMarkup(Loc.GetString("defusable-examine-live", ("name", uid),
+                    ("time", MathF.Floor(activeComp.TimeRemaining))));
+            }
+            else
+            {
+                args.PushMarkup(Loc.GetString("defusable-examine-live-display-off", ("name", uid)));
+            }
+        }
+        else
+        {
+            args.PushMarkup(Loc.GetString("defusable-examine-inactive", ("name", uid)));
+        }
+    }
+
+    #endregion
+
+    #region Anchorable
+
+    private void OnAnchorAttempt(EntityUid uid, DefusableComponent component, AnchorAttemptEvent args)
+    {
+        CheckAnchorAttempt(uid, component, args);
+    }
+
+    private void OnUnanchorAttempt(EntityUid uid, DefusableComponent component, UnanchorAttemptEvent args)
+    {
+        CheckAnchorAttempt(uid, component, args);
+    }
+
+    private void CheckAnchorAttempt(EntityUid uid, DefusableComponent component, BaseAnchoredAttemptEvent args)
+    {
+        // Don't allow the thing to be anchored if bolted
+        if (component.Bolted)
+        {
+            var msg = Loc.GetString("defusable-popup-cant-anchor", ("name", uid));
+            _popup.PopupEntity(msg, uid, args.User);
+
+            args.Cancel();
```

Comment:
dont cancel here, make this a bool and then in the handlers do something like 
if (CheckAnchorAttempt(uid, component, args))
	args.Cancel();